wandb/sdk/wandb_settings.py

```diff
@@ -455,14 +467,34 @@ def _default_props(self) -> Dict[str, Dict[str, Any]]:
         Note that key names must be the same as the class attribute names.
         """
         return dict(
+            _console={"hook": lambda _: self._convert_console(), "auto_hook": True},
```

Comment:
i see that you need the `_` in lambda because of the way you apply the hooks, right?